packages/user-interface/src/pages/CasesPage.test.tsx

```diff
@@ -1,8 +1,51 @@
 import { describe, it, expect } from "vitest";
-import CasesPage from "./CasesPage";
+import { MockCasesPage } from "./CasesPage.mock";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
 
-describe("CasesPage", () => {
-  it("is truthy", () => {
-    expect(CasesPage).toBeTruthy();
+describe("The CasesPage", () => {
+  let tabLopendeZaken = () =>
+    screen.getByRole("tab", { name: "Current cases" });
+  let tabAfgerondeZaken = () =>
+    screen.getByRole("tab", { name: "Closed cases" });
+  let getAllBezwaarCases = () => screen.getAllByText("case.B0756.title");
```

Comment:
Dit zijn Den Haag specifieke zaaktypes, die teksten staan ook enken in de dh-nl-portal-frontend, dus deze testen voor bezwaar zaken en Opas zaken horen hier niet thuis.

Reply:
Ik heb het aangepast en samengevoegd